src/data/helper.js

```diff
@@ -0,0 +1,53 @@
+export const calculateTotals = (daily) => {
```

Comment:
How about name this file something different than helper.js? This only contains a calculation of totals, how about just called it calculateTotals.js ?